_guides/getting_started.md

```diff
@@ -57,64 +86,140 @@ best constant = 0.500000
 best constant's loss = 0.250000
 total feature number = 15
 ```
-## VW's diagnostic information
 
-There is a burble of diagnostic information which you can turn off with `--quiet`. However, it's worthwhile to first understand it, so let's work through it bit by bit.
+## Vowpal Wabbit diagnostics
+
+This section provides information on the various types of diagnostic information VW presents.
+
+**Use** `--quiet`  command to turn off diagnostic information in Vowpal Wabbit.
+
+### Hash function bits
+
+The following VW output shows the number of bits from the hash function:
 
 ```
 Num weight bits = 18
 ```
-Only 18 bits of the hash function will be used. That's much more than necessary for this example. You could adjust the number of bits using `-b bits`
+
+This diagnostic shows that the number of bits from the hash function is 18 (more than enough for this example). Use `-b bits` to adjust the number of bits for a hash function.
+
+### Learning rate
+
+The following VW output shows the learning rate:
 
 ```
 learning rate = 0.5
 ```
-The default learning rate is 0.5 which we found to be a good default with the current default update (`--normalized --invariant --adaptive`). If the data is noisy you'll need a larger data-set and/or multiple passes to predict well. On these larger data-sets, our learning rate will by default decay towards 0 as we run through examples. You can adjust the learning rate up or down with `-l rate`. A higher learning rate will make the model converge faster but a too high learning rate may over-fit and end-up worse on average.
+
+The default learning rate is `0.5` with current default update (`--normalized --invariant --adaptive`). 
+
+If the data is noisy, you need a larger data-set or multiple passes to predict well. For massive data-sets, the learning rate decays to `0` by default. 
+
+**Use** `-l rate` to adjust the learning rate up or down.
+
+.**Note:** A high learning rate makes the model converge faster than a low learning rate, but if you adjust the learning rate too high, you risk over-fit and end-up worse on average.
+
+### Learning rate decay
+
+The following VW output sample shows the initial time:
 
 ```
 initial_t = 0
 ```
-Learning rates should often decay over time, and this specifies the initial time. You can adjust with `--initial_t time`, although this is rarely necessary these days.
+
+This diagnostic shows the initial time duration for learning rate decay. 
+
+**Use** `--initial_t time` to adjust the initial time. 
+
+>**Note:** It is common to need to adjust the initial time in Vowpal Wabbit. 
```

Comment:
@kumpera comments:
We're talking about options that the user might need to use with VW to achieve their goal.
Saying that it's not common doesn't tell anything about why.

Please confirm accuracy of statement now, "it IS common TO NEED to adjust `--initial_t time`...